master/usage/configuration/bgp.md

```diff
@@ -190,26 +230,33 @@ spec:
 EOF
 ```
 
-To remove a BGP peer that was configured with IP address aa:bb::ff (the AS
-number is not required), peering with {{site.prodname}} node "node1", run the following
-command from any node:
-
-	$ calicoctl delete bgpPeer aa:bb::ff --scope=node --node=node1
-
 To view the current list of node specific peers that are configured to peer
```

Comment:
we don't have a way to get all the node-specific bgppeers for a specific node anymore, so I think we can remove this whole section. Line 233 to 253

Reply:
I just reworded it to be more accurate. Thanks for the info!